### PR TITLE
Removed a null check to let null replace existing filehandleIds

### PR DESCRIPTION
**Pull request by Njanderson**

### CODE REVIEW

File: lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/UserProfileUtils.java [hunk 1]
```diff
@@ -31,8 +31,11 @@ public static void copyDtoToDbo(UserProfile dto, DBOUserProfile dbo) throws Data
 		} catch (IOException e) {
 			throw new RuntimeException(e);
 		}
-		if(dto.getProfilePicureFileHandleId()!= null){
-			dbo.setPictureId(Long.parseLong(dto.getProfilePicureFileHandleId()));
+		String profilePictureFileHandleId = dto.getProfilePicureFileHandleId();
```
Comment (jay-hodgson) [inline lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/UserProfileUtils.java:34]: Please add test (verifies that you can now set dbo picture id to null)
